src/org/exist/source/SourceFactory.java

```diff
@@ -177,15 +177,42 @@ else if (location.startsWith("file:/") || !location.contains(":/")) {
 
             if (source == null) {
                 /*
-                 * Lastly we try to load it using EXIST_HOME as the reference point
+                 * Try to load from the contextPath URL folder
```

Comment:
Is this safe (enough)? I can't estimate the risks...

Reply:
Should be. All the tests pass

Reply:
I meant: this adds file loading outside EXIST_HOME right? Will this change able ppl to break out an load e.g.  `/etc/shadow` ?

Reply:
We already had external loading before.  But none of it will parse as an XQuery do it won't load